plugins/_getdir/getdirs.php

```diff
@@ -157,38 +150,40 @@ function menuDblClickAndExit(obj)
 	menuClick(obj);
 	hideFrame();
 }
-
 </script>
 </head>
 <body onLoad='init()'>
-
-<table class='rmenuobj' cellpadding=0 cellspacing=0 width=100%>
-<?php
-function ordutf8($s) 
-{
-	if(function_exists("mb_convert_encoding"))
-	{
-		list(, $ret) = unpack('N', mb_convert_encoding(mb_strtolower($s), 'UCS-4BE', 'UTF-8'));
-	}
-	else
-	{
-		$ret = ord( strtolower($s) );
-	}
-	return($ret);
-}
-
-foreach($files as $key=>$data)
-{
-	$key = trim($key);
-	$chr = ordutf8($key);
-	if($key==='.')
-		echo "<tr><td code='".rawurlencode($data)."' id='root' class='rmenuitemselected' nowrap onclick='menuClick(this); return false;' ondblclick='menuDblClickAndExit(this); return false;'>";
-	else
-		echo "<tr><td code='".rawurlencode($data)."' class='rmenuitem' id='i".$chr."' nowrap onclick='menuClick(this); return false;' ondblclick='menuDblClick(this); return false;'>";
-	echo "&nbsp;&nbsp;";
-	echo $key;
-	echo "</td></tr>";
-}
-?>
-</table>
+	<div class="dir-list">
+		<div class="search-bar"><input type="text" class="filter-dir" placeholder="Type to filter..." /></div>
```

Comment:
Can we assign an id to this input type and focus it when the directory list iframe is created? This would prevent the user from having to click onto it to start typing.